php/elFinderVolumeGoogleDrive.class.php

```diff
@@ -0,0 +1,1659 @@
+<?php
+elFinder::$netDrivers['googledrive'] = 'GoogleDrive';
+
+/**
+ * Simple elFinder driver for GoogleDrive
+ * google-api-php-client-2.x or above
+ *
+ * @author Dmitry (dio) Levashov
+ * @author Cem (discofever)
+ **/
+class elFinderVolumeGoogleDrive extends elFinderVolumeDriver {
+
+	/**
+	 * Driver id
+	 * Must be started from letter and contains [a-z0-9]
+	 * Used as part of volume id
+	 *
+	 * @var string
+	 **/
+	protected $driverId = 'gd';
+
+	/**
+	 * OAuth object
+	 *
+	 * @var oauth
+	 **/
+	protected $oauth = null;
+	
+	//protected $client = null;
+
+	/**
+	 * GoogleDrive object
+	 *
+	 * @var googledrive
+	 **/
+	protected $googledrive = null;
+	
+    /**
+     * MIME tyoe of directory
+     *
+     * @var string
+     */
+    const DIRMIME = 'application/vnd.google-apps.folder';
+	
+	 /**
+     * Fetch fields for list
+     *
+     * @var string
+     */
+    const FETCHFIELDS_LIST = 'files(id,name,mimeType,modifiedTime,parents,size,imageMediaMetadata(height,width),webContentLink,thumbnailLink),nextPageToken';
+	
+	/**
+     * Fetch fields for get
+     *
+     * @var string
+     */
+    const FETCHFIELDS_GET = 'id,name,mimeType,modifiedTime,parents,size,imageMediaMetadata(height,width),webContentLink,thumbnailLink'; 
+	
+	/**
+	 * Directory for meta data caches
+	 * If not set driver not cache meta data
+	 *
+	 * @var string
+	 **/
+	protected $metaCache = '';
+	
+	/**
+	 * Last API error message
+	 *
+	 * @var string
+	 **/
+	protected $apiError = '';
+
+	/**
+	 * Directory for tmp files
+	 * If not set driver will try to use tmbDir as tmpDir
+	 *
+	 * @var string
+	 **/
+	protected $tmp = '';
+	
+	/**
+	 * Net mount key
+	 *
+	 * @var string
+	 **/
+	public $netMountKey = '';
+	
+	/**
+	 * drive.google.com uid
+	 *
+	 * @var string
+	 **/
+	protected $googledriveUid = '';
+	
+	/**
+	 * Gdrive download host, replaces 'drive.google.com' of shares URL
+	 * 
+	 * @var string
+	 */
+	private $googledrive_dlhost = 'drive.google.com';
+		
+	private $tmbPrefix = '';
+	
+	
+	/**
+	 * Constructor
+	 * Extend options with required fields
+	 *
+	 * @return void
+	 * @author Dmitry (dio) Levashov
+	 * @author Cem (DiscoFever)
+	 **/
+	public function __construct() {	
+		@ include_once 'google-api-php-client/vendor/autoload.php';
```

Comment:
Please remove this line. You must be add an option for google-api-php-client auto loader.